app/models/manageiq/providers/openstack/cloud_manager/refresh_parser.rb

```diff
@@ -25,7 +25,7 @@ def initialize(ems, options = nil)
       @network_service            = @os_handle.detect_network_service
       @image_service              = @os_handle.detect_image_service
       @volume_service             = @os_handle.detect_volume_service
-      @storage_service            = @os_handle.detect_storage_service
+      @storage_service            = @os_handle.detect_storage_service("admin")
```

Comment:
@roliveri heya, are you ok with this? It's because there would be an unscoped API query to swift. For some reason detect_storage_service contains query to directories.